Wire/ValueSerializers/CharSerializer.cs

```diff
@@ -16,7 +16,7 @@ public CharSerializer() : base(Manifest, () => WriteValueImpl, () => ReadValueIm
         public static char ReadValueImpl(Stream stream, byte[] bytes)
         {
             stream.Read(bytes, 0, Size);
-            return (char) BitConverter.ToSingle(bytes, 0);
```

Comment:
it's very strange https://msdn.microsoft.com/en-us/library/system.bitconverter.tosingle(v=vs.110).aspx